lab_1_classify_by_unigrams/start.py

```diff
@@ -14,7 +15,9 @@ def main() -> None:
         de_text = file_to_read_de.read()
     with open("assets/texts/unknown.txt", "r", encoding="utf-8") as file_to_read_unk:
         unknown_text = file_to_read_unk.read()
```

Comment:
you should assign `result = your last important function` for example: `result = tokenize(text)`

Reply:
isn't that what I did?
my last fuction is 'detect_language', which takes in language profiles

Reply:
I think I'm done?